.circleci/trigger_circleci.py

```diff
@@ -35,6 +35,8 @@ def main():
     }
     data.update(target)
 
+    print(f"Full data: {data}")
```

Comment:
any chance of this leaking secrets?

Reply:
no - this is from github event and github auto hides.

but will remove when this is debugged.